package.json

```diff
@@ -37,6 +37,7 @@
     "event-emitter-es6": "^1.1.5",
     "logdown": "^3.3.1",
     "reconnecting-websocket": "^4.4.0",
+    "rich-text": "^4.1.0",
```

Comment:
Is this dependency still needed? I don't see it being imported anywhere

Reply:
No, it was supposed to be removed. I guess I forgot to remove it here.

Reply:
Resolved in 18964cfa10dac2bd824d2bfffca128646e17fd8f